Source/GoBot/ViewDatabase+Pagination.swift

```diff
@@ -157,15 +157,17 @@ class RecentViewKeyValueSource: KeyValueSource {
 
     // home or explore view?
     private let onlyFollowed: Bool
-
-    init(with vdb: ViewDatabase, onlyFollowed: Bool = true) throws {
+    private let onlyRoots: Bool
+    
+    init(with vdb: ViewDatabase, onlyFollowed: Bool = true, onlyRoots: Bool = true) throws {
         self.view = vdb
         self.total = try vdb.statsForRootPosts(onlyFollowed: onlyFollowed)
```

Comment:
onlyRoots should also be used to compute the total number of posts (and in this case, comments as well)

Reply:
done